connect-examples/v2/node_subscription/package-lock.json

```diff
@@ -1,3004 +1,8 @@
 {
   "name": "square-subscription-api-sample-app",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Surprised this decremented

Reply:
this probably is set by the npm version of whoever generated this file.